dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -6,7 +6,7 @@ class Api::V1::UsersController < ApplicationController
 
   def load_user
     user_id = params[:user_id]
-    if user_id != 'me' && user_id != current_user.id
```

Comment:
@pcardune Not sure whether this was working before, but in my testing `user_id` was a `String` while `current_user.id` was a `Fixnum` and the comparison was not succeeding.